lib/src/main/kotlin/com/swmansion/starknet/helpers/Tip.kt

```diff
@@ -0,0 +1,49 @@
+@file:JvmName("Tip")
```

Comment:
note: It's needed for Java, so the import path is `import static com.swmansion.starknet.helpers.Tip.estimateTip;` instead of `import static com.swmansion.starknet.helpers.TipKt.estimateTip;`